wsagent/agent/src/main/resources/org.eclipse.che.ws-agent.script.sh

```diff
@@ -55,7 +55,7 @@ MACHINE_TYPE=$(uname -m)
 
 mkdir -p ${CHE_DIR}
 ${SUDO} mkdir -p /projects
-${SUDO} sh -c "chown -R $(id -u -n) /projects"
+${SUDO} sh -c "find /projects -not -uid $(id -u) -print0 | xargs -0 -P3 --no-run-if-empty chown $(id -u)"
```

Comment:
Looks like `find -uid` and `xargs -P3` are not POSIX compliant and not available in certain Linux distributions, such as alpine. 
@skabashnyuk @tolusha Do you think guys it's ok or should we explore this?

Reply:
on alphine 
```
/ # sh -c "find /projects -not -uid $(id -u) -print0 | xargs -0 -P3 --no-run-if-empty chown $(id -u)"
xargs: unrecognized option: P
find: unrecognized: -uid
BusyBox v1.26.2 (2017-06-11 06:38:32 GMT) multi-call bina
```

Reply:
Hmm... I reject it by myself and find the another way...